src/appleseed/foundation/image/genericimagefilewriter.cpp

```diff
@@ -297,6 +297,13 @@ void GenericImageFileWriter::set_exr_image_attributes(const ImageAttributes& ima
     }
 }
 
+void GenericImageFileWriter::set_bmp_image_attributes(const ImageAttributes& image_attributes)
+{
+    OIIO::ImageSpec& spec = impl->m_spec.back();
+
+    spec.attribute("ResolutionUnit", "m");
```

Comment:
What does this do? Why is it necessary?

Reply:
![bmp_oiio](https://user-images.githubusercontent.com/35020024/53680847-da1de080-3d06-11e9-9145-1fdf00616bc4.png)
It says in the oiio docs that for bmp ResolutionUnit is always m.
I think ResolutionUnit matters only when we print the image.
![resolutionunit_oiio](https://user-images.githubusercontent.com/35020024/53680961-5c5ad480-3d08-11e9-8666-916a21452da2.png)

Reply:
I'm not sure to understand. What in the docs makes you think you need to set `ResolutionUnit` to `m`? To me it seems that for the BMP format, it's always set to `m` anyway.